.jscsrc

```diff
@@ -0,0 +1,16 @@
+{
+  "maximumLineLength": 250,
+  "disallowDanglingUnderscores": { 
+  	"allExcept": [
+  		"_$http_", 
+  		"_formUrl", 
+  		"_generateListItems", 
+  		"_$compile_", 
+  		"_$rootScope_", 
+  		"_$controller_", 
+  		"_images_", 
+  		"_$q_", 
+  		"_$log_"
+  	] 
```

Comment:
The `disallowDanglingUnderscores` rule seems unnecessary, only because I've never had to do it in other projects. I've looked at other `.jscsrc` files and the other difference I see in those that I don't see here is a `preset` value. I've looked at the **airbnb** present settings and it does *not* include disallowDanglingUnderscores, so I'm a bit lost as to what setting allows Angulars underscore injections to work properly without explicitly setting them here.

@obuckley-kenzan Thoughts?